app/assets/javascripts/multiselect/index.js

```diff
@@ -1,2 +1,2 @@
-//= require knockout
+//= require knockout.js
```

Comment:
when does something need .js on the end vs not?

Reply:
I don't think it needs this? Likely the PR I pulled it from just had it